Task 3/Task 3.3/Task 3.3/PizzaTime/Order.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading;
+
+namespace Task_3._3
+{
+    class Order
+    {
+        Pizza pizza;
+        Thread tread;
+
+       
+        public Pizza Pizza { get => pizza;}
+
+        public event Action<Order> OnReady;
+        public Order(Pizza pizza)
+        {
+            this.pizza = pizza;  
+            tread = new Thread(Cocing);    
+        }
+        public void StartCocing()
+        {
+            tread.Start();
+        }
+        void Cocing()
+        {
+            Thread.Sleep(Pizza.CocingTime);
+            OnReady.Invoke(this);
```

Comment:
OnReady delegate can be empty (if there are no subscribers - you'll get NullReferenceException